fix(docs/backups): clarify file level restore for VG based on multiple PV (#6152)
vatesfr/xen-orchestra: docs/backups.md

File level restore **is only possible on delta backups**. Also, due of some technical limitations, you won't be able to do file level restore if you have a chain longer than 99 (ie retention longer than 99 records without any full between). Take a look at the [full backup interval section](./delta_backups.md#full-backup-interval) to set this correctly.
:::

:::warning

> File level restore **is only possible if all the partition is in the same VDI**. It excludes file recovery on VDIs of Volume Group based on multiple Physical Volumes.

Reviewer: ```suggestion
File level restore **is only possible on a single VDI**, it does not support LVM Volume Groups on multiple VDIs.
```
